app/validators/answers/option_answer_validator.py

```diff
@@ -68,3 +102,38 @@ def validate_default_exists_in_options(self):
             option["value"] for option in self.options
         ]:
             self.add_error(self.ANSWER_DEFAULT_MISSING, default_value=default_value)
+
+    def validate_dynamic_options(self):
```

Comment:
There is no one checking for return value of this method ?  you can just `return`

Reply:
It because the validate methods populates the object's errors attribute. It doesn't return the errors. This is consistent with our other validate methods.

Reply:
I think i was not clear,  what i meant on line 108 you are returning None , you can drop `None` just use `return`.  If there is no explicit check for `None` there is no point in returning `None`

Reply:
As discussed, this was to return it explicitly. I know we don't have mypy here, but this would be consistent with runner, as implicit optional is disabled there. But I don't mind, either. Do you want me to change?

Reply:
should be good as it is if it is consistent with runner